test/src/e2e_vm_tests/test_programs/should_pass/stdlib/assert_eq/Forc.toml

```
@@ -0,0 +1,8 @@
+[project]
+authors = ["Nicholas Furfaro"]
```

```suggestion
authors = ["Fuel Labs <contact@fuel.sh>"]
```

Should this be updated?